src/server/schema/resolvers/types/Life.ts

```diff
@@ -0,0 +1,14 @@
+import { GraphQLLifeResolvers } from '../definitions';
+
+const LifeGraphQL: GraphQLLifeResolvers = {
+    id: root => root._id,
+    firstName: root => root.firstName,
+    lastName: root => root.lastName,
+    fullName: root => `${root.firstName} ${root.lastName}`,
+    birthday: root => root.birthday,
+    hobbies: root => root.hobbies,
+    title: root => root.title,
+    description: root => root.description,
+};
```

Comment:
you dont have to repeat for those simple properties ( the properties has the same structure & name as its graphql counter part )

think here only need for `fullName` & `id`